src/shared/exec-util.c

```diff
@@ -283,16 +285,17 @@ int execute_strv(
 
         r = finish_serialization_fd(fd);
         if (r < 0)
-                return log_error_errno(r, "Failed to finish serialization fd: %m");
+                return log_error_errno(r, "Failed to finish serialization fd for %s: %m", name);
```

Comment:
doesn't matter, but I try to enclose user controlled strings in '' these days in all log messages, might be cool here too to write `'%s'` rather than `%s`.

one day we should update the conding style about this, making some weak suggestions around this.

Reply:
Same here, the string is not user-controlled, it is fixed in all callers.